cmd/buildah/addcopy.go

```diff
@@ -49,7 +50,11 @@ func addAndCopyCmd(c *cli.Context, extractLocalArchives bool) error {
 		dest = c.String("dest")
 	}
 	if name == "" && root == "" && link == "" {
-		return fmt.Errorf("either --name or --root or --link, or some combination, must be specified")
+		if len(args) == 0 {
+			return fmt.Errorf("either a container name or --root or --link, or some combination, must be specified")
+		}
+		name = args[0]
+		args = args.Tail()
```

Comment:
You've this same check in a few places scattered about.  Is it possible to put them into one common function?